WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -688,8 +707,7 @@ private extension StoreStatsV4PeriodViewController {
     }
 
     func hasRevenue() -> Bool {
-        let totalRevenue = orderStatsIntervals.map({ $0.revenueValue }).reduce(0, +)
-        return totalRevenue > 0
+        return revenueItems.map({ $0 != 0 }).contains(true)
```

Comment:
We can save a loop by using `contains(where:)`

```suggestion
        revenueItems.contains { $0 != 0 }
```